<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="5325" started-at="2020-01-06T15:25:29 IST" finished-at="2020-01-06T15:25:34 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="5325" started-at="2020-01-06T15:25:29 IST" finished-at="2020-01-06T15:25:34 IST">
      <class name="TestCases.HomeTestCase">
        <test-method status="FAIL" signature="verifyHomePageTC()[pri:0, instance:TestCases.HomeTestCase@ef309d]" name="verifyHomePageTC" duration-ms="133" started-at="2020-01-06T15:25:29 IST" finished-at="2020-01-06T15:25:29 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at commonProperties.Utilities.checkElementPresent(Utilities.java:83)
at PageObj.HomePage.verifyhomePage(HomePage.java:9)
at TestCases.HomeTestCase.verifyHomePageTC(HomeTestCase.java:12)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomePageTC -->
      </class> <!-- TestCases.HomeTestCase -->
      <class name="TestCases.LoginTestCase">
        <test-method status="PASS" signature="BrowsersetUp()[pri:0, instance:TestCases.LoginTestCase@b494a6]" name="BrowsersetUp" is-config="true" duration-ms="20755" started-at="2020-01-06T15:25:08 IST" finished-at="2020-01-06T15:25:29 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowsersetUp -->
        <test-method status="PASS" signature="chkLoginFunc()[pri:0, instance:TestCases.LoginTestCase@b494a6]" name="chkLoginFunc" duration-ms="5078" started-at="2020-01-06T15:25:29 IST" finished-at="2020-01-06T15:25:34 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- chkLoginFunc -->
      </class> <!-- TestCases.LoginTestCase -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
